chain-impl-mockchain/src/fee.rs

```diff
@@ -5,7 +5,7 @@ use std::num::NonZeroU64;
 
 /// Linear fee using the basic affine formula
 /// `COEFFICIENT * bytes(COUNT(tx.inputs) + COUNT(tx.outputs)) + CONSTANT + CERTIFICATE*COUNT(certificates)`.
-#[derive(PartialEq, Eq, PartialOrd, Debug, Clone, Copy)]
+#[derive(PartialEq, Eq, PartialOrd, Debug, Clone)]
```

Comment:
why?

Reply:
I am not in strong position with this change, but IMO `LinearFee` seems a complex structure with a lot of fields, so I would prefer to explicitly `clone` it where it is needed, so to have a better view how we can optimize our code.